chain/consensus/mir/state_manager.go

```diff
@@ -685,9 +696,22 @@ func (sm *StateManager) releaseNextCheckpointChan() {
 	}
 }
 
+func (sm *StateManager) waitForHeightWithTimeout(height abi.ChainEpoch) error {
+	log.With("validator", sm.id).Infof("waitForHeight %v started", height)
+	defer log.With("validator", sm.id).Infof("waitForHeight %v finished", height)
```

Comment:
I am afraid these logs may be a bit too chatty. Reading the logs of the validator these days is becoming increasingly hard. Maybe add them aas debug?

Reply:
fixed